tests/lldp/test_lldp.py

```diff
@@ -58,6 +58,8 @@ def test_lldp_neighbor(duthosts, enum_rand_one_per_hwsku_frontend_hostname, loca
     if tbinfo["topo"]["type"] != "t2":
         mgmt_alias = duthost.get_extended_minigraph_facts(tbinfo)["minigraph_mgmt_interface"]["alias"]
         switch_mac = duthost.get_dut_iface_mac(mgmt_alias)
+    elif tbinfo["topo"]["type"] == "t2":
+        switch_mac = config_facts['DEVICE_METADATA']['localhost']['mac'].lower()
     else:
         switch_mac = duthost.facts['router_mac']
```

Comment:
@vperumal ,  in the else block `duthost.facts['router_mac']` gets the router mac from the config_db
https://github.com/Azure/sonic-mgmt/blob/cd8c365c6626720102695b8f8dc052eddcd32df2/tests/common/devices/sonic.py#L240-L242

This doesnt work ? Curious why we need this change

Reply:
@arlakshm  this is host level router-mac. For Chassis-packet each asic has it's unique router-mac.